### PR TITLE
[SP-4809] Adding AWS output examples

### PR DESCRIPTION
**Pull request by igorkheif**
* Bunch of output examples for the "raw" aws cli manifests
* Tiny code change in get-ips-for-instances
* Did not touch redshift since it's already well documented

### CODE REVIEW

File: steps/aws/attach-volume/manifest.yaml [hunk 1]
```diff
@@ -34,5 +34,27 @@ args:
     required: true
     default: "/dev/sdf"
 
+outputs:
+  - name: AttachTime
```
Comment (hagaishapira) [inline steps/aws/attach-volume/manifest.yaml:38]: The output json has capitalized keys?

File: steps/aws/describe-instances/manifest.yaml [hunk 2]
```diff
@@ -23,5 +23,11 @@ args:
...
...","DeleteOnTermination":true,"DeviceIndex":0,"Status":"attaching"},"Description":"","Groups":[{"GroupName":"MySecurityGroup","GroupId":"sg-0598c7d356eba48d7"}],"Ipv6Addresses":[],"MacAddress":"0a:ab:58:e0:67:e2","NetworkInterfaceId":"eni-0c0a29997760baee7","OwnerId":"123456789012","PrivateDnsName":"ip-10-0-0-157.us-east-2.compute.internal","PrivateIpAddress":"10.0.0.157","PrivateIpAddresses":[{"Primary":true,"PrivateDnsName":"ip-10-0-0-157.us-east-2.compute.internal","PrivateIpAddress":"10.0.0.157"}],"SourceDestCheck":true,"Status":"in-use","SubnetId":"subnet-04a636d18e83cfacb","VpcId":"vpc-1234567890abcdef0","InterfaceType":"interface"}],"RootDeviceName":"/dev/xvda","RootDeviceType":"ebs","SecurityGroups":[{"GroupName":"MySecurityGroup","GroupId":"sg-0598c7d356eba48d7"}],"SourceDestCheck":true,"StateReason":{"Code":"pending","Message":"pending"},"Tags":[],"VirtualizationType":"hvm","CpuOptions":{"CoreCount":1,"ThreadsPerCore":1},"CapacityReservationSpecification":{"CapacityReservationPreference":"open"},"MetadataOptions":{"State":"pending","HttpTokens":"optional","HttpPutResponseHopLimit":1,"HttpEndpoint":"enabled"}}],"OwnerId":123456789012,"ReservationId":"r-02a3f596d91411712"}]'
```
Comment (hagaishapira) [inline steps/aws/describe-instances/manifest.yaml:30]: are you sure the escaping is correct, looks like github is failing to syntax highlight this

**Review by hagaishapira — APPROVED** [review]
See comments

File: steps/aws/attach-volume/manifest.yaml (hunk 1 above)
Comment (igorkheif) [reply steps/aws/attach-volume/manifest.yaml:38]: Yeah
image

File: steps/aws/describe-instances/manifest.yaml (hunk 2 above)
Comment (igorkheif) [reply steps/aws/describe-instances/manifest.yaml]: Weird. Got no complaints from neither Goland nor VSCode. Anyway, changed it a bit to fit github. Looks ok now.

**Review by InbarPerry — APPROVED** [review]

**Pull request merged**